packages/esm-patient-common-lib/src/get-patient-uuid-from-url.ts

```diff
@@ -0,0 +1,4 @@
+export function getPatientUuidFromUrl(): string {
```

Comment:
I made use of this function a lot. Not sure if this is consistent with how we want to handle this in the patient chart generally. Obviously is a different approach from `usePatient` etc.

Reply:
Well, it's quite specific to the patient chart URL

Reply:
Yes, it's just also the function we use in `usePatient()` if a patient UUID isn't supplied, but I don't disagree that it's very patient-chart specific.